src/index.html

```diff
@@ -9,9 +9,191 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
+    <script type="text/javascript" src="./main.js"></script>
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <label class="label">
+          Surname:
+
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            required
+            pattern="\w+"
+          >
+        </label>
+
+        <label class="label">
+          Name:
+
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            required
+            pattern="\w+"
+          >
+        </label>
+
+        <label class="label">
+          How old are You?
+
+          <input
+            type="number"
+            name="age"
+            placeholder="Your age"
+            value="12"
+            min="1"
+            max="100"
+            required
+          >
+        </label>
+
+        <label class="label">
+          Full date of birth:
+
+          <input
+            type="date"
+            name="date of birth"
```

Comment:
```suggestion
            name="dateOfBirth"
```
[checklist p.4](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)